enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/catchup/storecopy/StoreCopyClient.java

```diff
@@ -77,7 +71,7 @@ public boolean onFileContent( CompletableFuture<Long> signal, FileChunk fileChun
                         public void onFileStreamingComplete( CompletableFuture<Long> signal,
                                 StoreCopyFinishedResponse response )
                         {
-                            log.info( "Finished streaming %s", destination );
+                            log.info( "Finished streaming %s" );
```

Comment:
this will not log anything

Reply:
Sloppy removal, will remove %s as well. The destination was removed on purpose because that is already logged when receiving and is not interesting to log again on completion.